lib/pure/collections/deques.nim

```diff
@@ -10,19 +10,18 @@
 ## Implementation of a `deque`:idx: (double-ended queue).
 ## The underlying implementation uses a ``seq``.
 ##
-## None of the procs that get an individual value from the deque can be used
-## on an empty deque.
-## If compiled with `boundChecks` option, those procs will raise an `IndexError`
-## on such access. This should not be relied upon, as `-d:release` will
-## disable those checks and may return garbage or crash the program.
+## Deques are implicitly initialised as empty, similar to tables and seqs. But
```

Comment:
here (+elsewhere ?): initialized

Reply:
British English vs. American English, I probably have many more instances of this in my documentation..

Reply:
fine, not a big deal (but I usually settle with with majority vote looking at exact match on google, it gives 18M for initialized vs 1M for initialised)